content/docs/applicationmobility/_index.md

```diff
@@ -7,7 +7,7 @@ Description: >
 ---
 
 {{% pageinfo color="primary" %}}
-Application Mobility is available with [APEX Navigator for Kubernetes](https://www.dell.com/en-ca/dt/apex/storage/public-cloud/navigator.htm#kubernetes)
+Coming Soon: Application Mobility will be available with CSM early next year (2025)
```

Comment:
Change this to be more professional - "We are pleased to announce that Application Mobility will be available with Container Storage Modules starting early next year (2025)."